canonical-web-and-design/vanilla-framework
# Consistent hover background color

## Done

Make buttons, sidenav, contextual menu etc use the same hover color.

## QA

- Pull code
- Run `./run`
Paste the following in any example, inspect the hover states and make sure background color matches:

```
<div class="p-strip--light">
  <div class="row">
    <div class="col-6">
      <input type="search" name="" id="" placeholder="search" class="u-sv-2 u-no-margin--bottom">
      <div class="p-chips__chip--with-category"><span class="p-chips__category"></div>
        <button class="p-button--positive">Positive</button>
        <button class="p-button--negative">p-button--negative</button>
        <button>Hover state</button>
        <button>Active state</button><br>
      {% include "docs/examples/patterns/contextual-menu/default.html" %}
      {% include "docs/examples/patterns/accordion.html" %}
    </div>
    <div class="col-6">
      {% include "docs/examples/patterns/side-navigation/_default.html" %}
    </div>
  </div>
</div>
```

## Screenshots
![image](https://user-images.githubusercontent.com/2741678/83887152-f39aad00-a73f-11ea-80d6-a82285344314.png)
